app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,15 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  height: 70px;
+  border-top: 1px solid $govuk-border-colour;
+  margin-bottom: 16px;
```

Comment:
Is this intentional? It's added a gap between the elements, which seems unnecessary.

<img width="455" alt="Screenshot 2020-02-19 at 08 18 20" src="https://user-images.githubusercontent.com/861310/74815130-9134b280-52f0-11ea-801c-d195f97a350c.png">

Reply:
It is as per the design document, will check with mia

Reply:
feedback from Mia is it should be as designed, visual separation is useful on mobile devices

Reply:
with this extra padding on the first element it's inconsistent when accessed via keyboard